e2e/features/send-decision-and-reasons.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Send decision and reasons
+
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+    And I request hearing requirements
+    And I list the case
+    And I create case summary
+    And I generate the hearing bundle
+    And I start decision and reasons
+    And I generate decision and reasons
+
+  @send-decision-and-reasons-dismissed @RIA-1154
+  Scenario: Send decision and reasons with Dismissed status
```

Comment:
Is this not the scenario for Allowed?

Reply:
The first version contains checks for "Change" links in confirmation page where the decision was changed to dismissed.
So eventually Dismissed status was submitted for the case.

I think Tony removed that part, maybe it was not very clear what is going on.